source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Skydive With oVirt 
+author: mburman
+tags: oVirt, oVirt 4.2, open source virtualization,network
+date: 2018-07-10 09:01:00 UTC
+---
+
+## Benefit to oVirt users
+oVirt users may now analyze and see thier network topology via the Skydive UI. 
+Skydive allows the users to see the result of their configuration done via oVirt.
+They may capture traffic from VM1 to VM2 for example or monitor the traffic between two VMs or hosts. 
+Also they may generate some traffic between 2 running VMs on different hosts for example.
+Users may create alerts in Skydive UI to notify that some traffic disconnected or down.
```

Comment:
Suggest "Administrators can create alerts in the Skydive UI to notify when traffic is disconnected or down."

Reply:
done